modules/core/src/lib/layer-manager.js

```diff
@@ -177,9 +177,9 @@ export default class LayerManager {
   /* eslint-enable complexity, max-statements */
 
   // Supply a new layer list, initiating sublayer generation and layer matching
-  setLayers(newLayers) {
+  setLayers(newLayers, force) {
```

Comment:
This is not supposed to be called by layers or apps right? Should this be an underscore method? 

Also the passing in of `this.lastRenderedLayers` seems weird, looks like it always bails? Maybe this method can be made more clear, perhaps just pass in `null` if nothing new or similar.

Reply:
This is used by `@deck.gl/test-utils`, where `LayerManager` is used standalone.